packages/mask/src/social-network-adaptor/facebook.com/utils/user.ts

```diff
@@ -16,7 +17,9 @@ import { isMobileFacebook } from './isMobile'
 import { bioDescription, personalHomepage } from '../../../settings/settings'
 
 export const getNickName = () => {
-    const node = isMobileFacebook ? searchNickNameSelectorOnMobile().evaluate() : searchNickNameSelector().evaluate()
+    const node = isMobileFacebook
+        ? searchNickNameSelectorOnMobile().evaluate() || searchNickNameSelectorOnMobileBig().evaluate()
```

Comment:
Can we merge `searchNickNameSelectorOnMobileBig` into `searchNickNameSelectorOnMobile`?